### ISSUE TITLE
Hunter Summon Pet blocked by Hekili

### ISSUE THREAD

**Issue by derekengel** [Hekili/hekili#288]
On a hunter, dismiss or have no pet.  With Hekili enabled, try calling or summoning a pet.  When you do, a window comes up saying Blizzard blocked Hekili addon with a choice to disable or ignore.  Disable the addon, summon your pet, enable addon and everything is fine.

**Comment by Hekili** [comment]
Screenshots?

BugSack, BugGrabber?

**Comment by derekengel** [comment]
There is no bugsack or bugrabber that i can get.  I tried doing it again just now to take a screenshot of the pop-up, and it worked.  So I guess its random.  It has happened before though.  Again, just summon a pet with the addon enabled and you might get the Blizzard pop-up.

**Comment by Hekili** [comment]
If you get a screenshot of the message, feel free to submit a new ticket with the screenshot and all the info requested in the issue ticket.  Thanks!
